engine/client/gl_rstrobe.c

```diff
@@ -18,29 +32,34 @@ typedef enum
 {
 	TotalFrame,
 	PositiveFrame,
-	PositiveNormalFrame,
+	PositiveRenderedFrame,
 	PositiveBlackFrame,
 	NegativeFrame,
-	NegativeNormalFrame,
+	NegativeRenderedFrame,
 	NegativeBlackFrame
 } CountType;
 
 typedef enum
 {
 	PHASE_POSITIVE = BIT( 0 ),
 	PHASE_INVERTED = BIT( 1 ),
-	FRAME_RENDER   = BIT( 2 )
-} FrameState; // Frame State
+	FRAME_RENDER   = BIT( 2 ),
+
+	_unassigned1 = ( PHASE_POSITIVE | PHASE_INVERTED ),
```

Comment:
Any purpose?

Reply:
Without these, for example setting FrameSetting to PHASE_POSITIVE | PHASE_INVERTED causes warning as there is no such thing in the enum's definition. Maybe instead of using an enum use something else should be used ?

Reply:
Yeah, this warning means, that you're trying to put a bigger value than enum can hold. You may use just an integer, instead of enum type.